### PR TITLE
hide player controls while not playing the video

### LINKED ISSUE #97: Hide #playerControls if not currently playing a video
I don't know how I didn't notice this until now... we don't need to show the player controls at all unless we're playing a video.
For example: a when you first load up the page, you shouldn't see play/pause, show player, show description, or the volume controls because they're totally useless.

Solution:
- [ ] in `css/styles.css` add a new rule:
```css
#playerControls {
    display: none;
}
```
- [ ] in `js/everything.js` around line 365 under `if (currentVideoID) {` add
```javascript
$("#playerControls").show();
```

### PR DESCRIPTION
**Pull request by Ab1d**
this fixes #97

### CODE REVIEW

**Comment by shakeelmohamed** [comment]
Can you add a new line in `.gitignore` with `.DS_Store`?

**Comment by shakeelmohamed** [comment]
Can we move the `<br>` from line 40 in `index.html` to be directly inside `<div id="audioplayer">`?

**Comment by Ab1d** [comment]
yeah was thinking of the same, check now.

**Comment by shakeelmohamed** [comment]
Looks great! Since the `.DS_Store` file was already checked in, you'll need to remove the file (`git rm .DS_Store`) and make another commit.

File: css/styles.css [hunk 1]
```diff
@@ -34,6 +34,9 @@ header, footer {
     width: 100%;
     display: none;
 }
+#playerControls {
+    display:none;
```
Comment (shakeelmohamed) [inline css/styles.css]: nit: can we add a space after the colon?

**Comment by shakeelmohamed** [comment]
Perfect, thanks @Ab1d. Welcome to the org, and happy new year! Thanks for being the first contributor of 2016

**Comment by Ab1d** [comment]
Thanks @shakeelmohamed this is my first open source contribution :), Happy new year to you too.

**Comment by shakeelmohamed** [comment]
@Ab1d Awesome! I hope to see future contributions from you :smiley:

**Pull request merged**